### DISCUSSION TITLE
Next suggestion dutch mac keyboard

### DISCUSSION THREAD

**Discussion by charleskwakye** [community/community#14424] [Copilot Conversations]
How do I show next suggestion on the dutch keyboard o mac. Its (option + ])  on US keyboard but typing "]" on dutch keyboard requires two keys.

**Comment by Urethramancer** [comment]
Similar issues on many European keyboards. I changed the key for "Show Next Inline Suggestion", and that works perfectly for me. Try changing that to some handy combination. You may have to delete a shortcut you didn't know about, but in that case you probably don't need the old one :)
